plaso/storage/sqlite/merge_reader.py

```diff
@@ -9,17 +9,22 @@
 
 from plaso.containers import event_sources
 from plaso.containers import events
+from plaso.containers import manager as containers_manager
 from plaso.containers import reports
 from plaso.containers import tasks
 from plaso.containers import warnings
 from plaso.lib import definitions
 from plaso.storage import interface
 from plaso.storage import identifiers
+from plaso.storage import logger
 
 
 class SQLiteStorageMergeReader(interface.StorageMergeReader):
   """SQLite-based storage file reader for merging."""
 
+  _ATTRIBUTE_CONTAINERS_MANAGER = (
```

Comment:
This is misnamed? This a method on the manager, not the manager itself.

Reply:
ack